packages/core/src/secret/crypto-functions.ts

```
@@ -2,11 +1,7 @@
+import { createHash, pbkdf2 as pbkdf2Node } from 'crypto';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)